### DISCUSSION TITLE
Please help

### DISCUSSION THREAD

**Discussion by VasilyProfWork** [henrygd/bigger-picture#23] [Q&A] [answered]
Please tell me how I can use thumbnails on pure javaScript I don't understand

**Comment by henrygd** [comment, marked as answer]
Hi, you'd need to do your own implementation because it's not a part of the library. Just an example.

There are alternative libraries that support thumbnails by default so you could try one of them. I know [lightGallery](https://github.com/sachinchoolur/lightGallery) does, but be aware that it's not free for commercial projects.

For more info see this issue: https://github.com/henrygd/bigger-picture/issues/16

**Reply by VasilyProfWork** [reply]
Thank you very much
